workshop.wwwapi/Data/DatabaseContext.cs

```diff
@@ -17,10 +17,33 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             //TODO: Appointment Key etc.. Add Here
-            
+            modelBuilder.Entity<Appointment>().HasKey(e => new { e.Booking, e.PatientId, e.DoctorId, e.PrescriptionId });
```

Comment:
prescriptions should not be part of the primary key because they are optional: you can go visit a doctor and get no prescription for a medicine :) no need to change it though, you can keep your design